### DISCUSSION TITLE
Maintain Navigation Scroll Position

### DISCUSSION THREAD

**Discussion by discoverlance-com** [livewire/livewire#7316] [1. Help]
In the livewire documentation, I realised that after scrolling down in the navigation and clicking on a certain link, the scroll position resets to the top of the page. It will be very good if the sidebar or navigation scroll position could be maintained after clicking on a link but of course the main page content can be scrolled to the top to get the new page content that has changed. I think this can be very useful not only for livewire docs but for dashboards as I am building one too. Not sure how to implement it though.

[Screen recording 2023-11-19 19.43.31.webm](https://github.com/livewire/livewire/assets/52635726/3d632ac1-0baa-41b4-8517-80f445093749)

**Comment by pronode** [comment]
Does this solve your problem? https://livewire.laravel.com/docs/navigate#preserving-scroll-position
